guide/exporting.md

```diff
@@ -1,6 +1,8 @@
 # Exporting
 
-## PDF
+## Slides
```

Comment:
Since they are two different sections, I preferred to specify the `Slides` and the `Presenter notes` sections.